src/test/java/com/tom/driver/manager/DriverUtils.java

```diff
@@ -0,0 +1,11 @@
+package com.tom.driver.manager;
+
+public class DriverUtils {
+    public static void setInitialConfiguration(){
+        DriverManager.getWebDriver().manage().window().maximize();
+    }
+
+    public static void navigateToPage(String pageUrl){
```

Comment:
This method should be in BasePage class I think

Reply:
added new class PageURLs to keep links to test pages.